src/components/DataTypes/array.tsx

```diff
@@ -1,4 +1,4 @@
-import { JSONSchema6 } from 'json-schema';
+import { JSONSchema7 } from 'json-schema';
```

Comment:
We could import it like this `import { JSONSchema7 as JSONSchema } from 'json-schema';` to avoid lots of changes if we ever change it to `JSONSchema8`